manifests/init.pp

```diff
@@ -98,6 +102,14 @@
   Ferm::Chain <| |>
   ~> Class['ferm::service']
 
+  # collect chains from hiera
```

Comment:
hiera is not mandatory to use this. It can also come from simple puppet code.

Reply:
thanks for the clarification, removed the misleading comments